packages/transit-vehicle-overlay/src/index.story.js

```diff
@@ -106,7 +106,9 @@ function rectangles(popup = true) {
   // initial setup
   const vehicleData = line;
   const patternGeometry = utils.makePattern(geom, "111");
-  const recenter = utils.recenterFlyTo(null);
+
+  const doTracking = boolean("Recenter map on tracked vehcile", true);
```

Comment:
Maybe a better way to label this flag:
```suggestion
  const doTracking = boolean("Recenter map when tracking a different vehicle", true);
```